header.h

```diff
@@ -0,0 +1,144 @@
+enum Colors
+{
+    RED,
+    GREEN,
+    BLUE,
+    WHITE,
+};
+
+class Point
+{
+public:
+    int x;
+
+    int y;
+
+    int colors;
+
+    Point(){}
+
+    Point(int value_x, int value_y, Colors color)
+    {
+        colors = color;
+        x = value_x;
+        y = value_y;
+
+    }
+
+    int Getx()
+    {
+        return x;
+    }
+
+    int Gety()
+    {
+        return y;
+    }
+
+    int func1()
+    {
+        int Perumetr = 2 * (x + y);
+        return Perumetr;
+    }
+
+    double func2()
+    {
+        double diagonal = sqrt(pow(x, 2) + pow(y, 2));
+        return diagonal;
+    }
+
+    void Print()
+    {
+        cout << "X = " << x << "\t" << "Y = " << y << "\t";
+
+        cout << "Perumetr = " << func1() << "\t" << "Diagonal = " << func2() << endl;
+    }
+
+    void Painting() {
+        cout << endl << "    " << endl;
+        int** array = new int* [y];
+
+        for (int i = 0; i < y; i++)
+        {
+            array[i] = new int[x];
+        }
+
+        for (int i = 0; i < y; i++)
+        {
+            for (int j = 0; j < x; j++)
+            {
+                array[i][j] = colors;
+            }
+        }
+
+        for (int i = 0; i < y; i++)
+        {
+            for (int j = 0; j < x; j++)
+            {
+                cout << array[i][j] << "  ";
+            }
+            cout << endl;
+        }
+
+        for (int i = 0; i < y; i++)
+        {
+            delete[] array[i];
+        }
+        delete[] array;
+        cout << "    " << endl;
+    }
+
+    ~Point()
+    {
+
+    }
+};
+
+
+
+class Polynom
+{
+public:
+
+    Polynom(){}
+
+    Point array_of_points[99];
+    Point array_sorted[1];
+    int numberelements = 0;
+
+    void AddArray(Point point) {
+        array_of_points[numberelements] = point;
+        numberelements++;
+    }
+
+    void AllPointsByX() {
```

Comment:
void allPointsByX()
       {
        for (int i = 0; i < number_elements; i++) 
        {
            for (int j = 0; j < numb_erelements; j++)
                if (array_of_points[j].x < array_of_points[j + 1].x) 
             {
                    array_sorted[0] = array_of_points[j+1];
                    array_of_points[j+1] = array_of_points[j];
                    array_of_points[j] = array_sorted[0];
                }
        }
        for (int n = 0; n < number_elements; n++)
        {
            array_of_points[n]. printPointAndPerumetrAndDiagonalToConsole();
        }
       }